storm-core/test/clj/backtype/storm/nimbus_test.clj

```diff
@@ -1238,10 +1238,11 @@
   (testing "nimbus-data uses correct ACLs"
     (let [scheme "digest"
           digest "storm:thisisapoorpassword"
-          auth-conf {STORM-ZOOKEEPER-AUTH-SCHEME scheme
+          auth-conf (merge (read-storm-config)
+                    {STORM-ZOOKEEPER-AUTH-SCHEME scheme
                      STORM-ZOOKEEPER-AUTH-PAYLOAD digest
                      STORM-PRINCIPAL-TO-LOCAL-PLUGIN "backtype.storm.security.auth.DefaultPrincipalToLocal"
-                     NIMBUS-THRIFT-PORT 6666}
+                     NIMBUS-THRIFT-PORT 6666})
```

Comment:
All the test cases in this file have a similar port, will create a JIRA to tackle this issue separately as I feel it is not a part of this change